<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>index</class>
 <widget class="QWidget" name="index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>761</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Recommend 药品推荐</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QWidget" name="widget_3" native="true">
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>171</width>
           <height>111</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/pictures/med_pic/10.jpeg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>-50</x>
           <y>0</y>
           <width>351</width>
           <height>101</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">
image: url(:/pictures/med_pic/0.jpeg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>221</width>
           <height>101</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/pictures/med_pic/1.jpeg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_4" native="true">
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>0</y>
           <width>161</width>
           <height>101</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/pictures/med_pic/11.jpeg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_5" native="true">
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>231</width>
           <height>101</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/pictures/med_pic/5.jpeg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="widget_6" native="true">
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>0</y>
           <width>151</width>
           <height>101</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/pictures/med_pic/8.jpeg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>761</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_med">
        <property name="text">
         <string>药品管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_supplier">
        <property name="text">
         <string>供应商管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_purchase">
        <property name="text">
         <string>采购管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_person">
        <property name="text">
         <string>用户管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QAnimationStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>751</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>欢迎您， </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAnimationStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>qanimationstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
